sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/util/ExtendedRow.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.stargate.sgv2.docsapi.service.util;
+
+import io.stargate.grpc.Values;
+import io.stargate.proto.QueryOuterClass.ColumnSpec;
+import io.stargate.proto.QueryOuterClass.Row;
+import io.stargate.proto.QueryOuterClass.Value;
+import java.util.List;
+
+/** A wrapper around the protobuf-generated {@link Row}, to add the column metadata. */
+public class ExtendedRow {
```

Comment:
Maybe we should move this to `sgv2-service-common` and generalize its use. I've been passing the row and columns separately in GraphQL, but here it really makes sense to have a wrapper with all the predicates.

Reply:
Yea wrapper makes sense, few things:

* can this also be immutable
* can this be part of some kind of the `.model` package? Also the naming makes it confusing with the extended row we implemented on the bridge, but yea.. Maybe `RowWrapper` or something like this..